README.md

```diff
@@ -9,18 +9,13 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 This repository contains test results for TrenchBoot project for the following
 mainboards:
 
--
+The `boards` directory contains the current test results for specified board
 
-The `boards` directory contains the current test results retrieved from the
-[Dasharo Test & Feature Matrix](https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit#gid=736501945)
-sheet.
-
-The `archive` section contains archival test results retrieved from previous
-iterations of the test sheets and is preserved here for historical purposes.
+* [APU](./boards/APU/) - results for different PC Engines APU boards
 
 ## Links
```

Comment:
@m-iwanicki let's add reference to the Cukinia test code

Reply:
@artur-rs Added in [dae787a](https://github.com/TrenchBoot/test-results/pull/1/commits/dae787af8aeb762353cfddc14e22674029c25e41).
Added link with exact commit, after it's merged to master I could change it to always point to newest version on master.